src/transformers/models/maskformer/configuration_maskformer.py

```diff
@@ -0,0 +1,149 @@
+# coding=utf-8
+# Copyright 2021 Facebook AI Research and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" MaskFormer model configuration"""
+
+from dataclasses import dataclass, field
+from typing import Dict, List, Optional, Tuple
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+MASKFORMER_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    # "facebook/detr-resnet-50": "https://huggingface.co/facebook/detr-resnet-50/resolve/main/config.json",
+    # See all MaskFormer models at https://huggingface.co/models?filter=detr
+}
+
+logger = logging.get_logger(__name__)
+
+
+@dataclass
+class ClassSpec:
+    is_thing: bool
+    label: str
+    color: Tuple[int, int, int]
+
+
+@dataclass
+class DatasetMetadata:
+    classes: List[ClassSpec] = field(default_factory=list)
+
+
+class MaskFormerConfig(PretrainedConfig):
+    r"""
+    This is the configuration class to store the configuration of a [`MaskFormer`]. It is used to instantiate a
+    MaskFormer model according to the specified arguments, defining the model architecture. Instantiating a
+    configuration with the defaults will yield a similar configuration to that of the "askformer-swin-base-640"
```

Comment:
```suggestion
    configuration with the defaults will yield a similar configuration to that of the "maskformer-swin-base-ade-640"
```
Maybe we can call it like that.